setup.py

```diff
@@ -54,7 +54,7 @@
         'scipy',
         'Jinja2'
       ],
-      python_requires='>=2.7,!=3.1,!=3.2,!=3.3',
+      python_requires='>=2.7,!=3.1,!=3.2,!=3.3,!=3.4',
```

Comment:
Let's drop 2.7, and only do `>3.6`.  Does anyone use 3.5 anymore?

Reply:
Nope, 3.5 not supported.